src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/resource/AdminController.kt

```diff
@@ -60,6 +62,17 @@ class AdminController(
     return adminService.deactivateNomisOffence(offenceIds)
   }
 
+  @PostMapping(value = ["/nomis/offences/encouragement/{parentOffenceId}"])
+  @PreAuthorize("hasRole('ROLE_NOMIS_OFFENCE_ACTIVATOR')")
```

Comment:
is this the right role to use? there's a move to use more granular roles

Reply:
@joelstobart-moj any thoughts on this? The Admin controller uses the same permission for each action. I could update them to an alternative if you feel its worthwhile as part of the ticket?

The same changes would then be added to the frontend application.